### ISSUE TITLE
cannot be moved on to the next page

### ISSUE THREAD

**Issue by Caffeine21** [mongo-express/mongo-express-docker#104]
We found that the page move URL contains the variable runAggregate=false.
If this variable is included, the URL will not work.
I'm using the latest version. Is there a way to solve this problem? 

<img width="991" alt="image" src="https://github.com/mongo-express/mongo-express-docker/assets/64478772/d086226f-95af-41bc-bf20-4b1efde6f9a4">

**Comment by Caffeine21** [comment]
And what is this useless button on both sides..?

<img width="142" alt="image" src="https://github.com/mongo-express/mongo-express-docker/assets/64478772/4c73e0df-8063-4630-94d7-a5440aca0800">

**Comment by BlackthornYugen** [comment]
> We found that the page move URL contains the variable runAggregate=false.
> If this variable is included, the URL will not work.
> I'm using the latest version. Is there a way to solve this problem? 
> 
> <img width="991" alt="image" src="https://github.com/mongo-express/mongo-express-docker/assets/64478772/d086226f-95af-41bc-bf20-4b1efde6f9a4">

I think this is fixed in the latest npm version but it hasn't been published to docker yet due to an issue with the build.

**Comment by Caffeine21** [comment]
Then there's no way I can solve it.
When will it be updated?

**Comment by Caffeine21** [comment]
Thank you. Have a nice day today

**Comment by BlackthornYugen** [comment]
I think it makes sense to keep the issue open until it's fixed right?

**Comment by Caffeine21** [comment]
Okay.

**Comment by bharath-naik** [comment]
facing the same issue for latest mongo-express docker. is there any version of docker where paginantion issue is working.

**Comment by OsoianMarcel** [comment]
I have the same issue on the latest Docker image.

**Comment by nofacer** [comment]
same here. Which tag is able to use now?

**Comment by eggcho** [comment]
> same here. Which tag is able to use now?

[1.0.0-alpha.4] It works for me.

**Comment by Massoumeh-Mahaei** [comment]
Is it okay to use alpha image in production ?

**Comment by nidhinpmohan** [comment]
When we can expect the solution ? any specific date ?

**Comment by eviltik** [comment]
PLEASE WAAAAAAAAAAKEEEEEEEEEE UPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPP

**Comment by BlackthornYugen** [comment]
As I said in October, I'm not able to give a timeframe. I've tried a few times to get the build to work so I could update the docker image but I couldn't figure it out. I don't have a lot of time to put to this project. If someone is able to get the to get the docker build working I can publish the latest NPM version to docker.io.

**Comment by BlackthornYugen** [comment]
Changes in #107 / #108 seem to be working. I'm confident that once one of these PRs is merged, we will be able to publish 1.0.2. Thanks @fitztrev for your help!

**Comment by nidhinpmohan** [comment]
Pagination working with new tag 1.0.2-   Thank you.....

**Comment by eviltik** [comment]
Thank you !

**Comment by BlackthornYugen** [comment]
Sounds like it was resolved. 🙂

**Comment by Caffeine21** [comment]
Page button not created after page 5
It also saves well in the actual database and works well when changing the URL (skip=50), but the page 6 button will not be generated.

**Comment by bkis** [comment]
> Page button not created after page 5 It also saves well in the actual database and works well when changing the URL (skip=50), but the page 6 button will not be generated.

Can confirm. I had this issue with pagination ending at page 2. I have no idea what the exact number of generated pagination links depends on, though (I mean in the context of this bug).

**Comment by BlackthornYugen** [comment]
@rtritto is this fixed in the [1.0.3-rc](https://github.com/mongo-express/mongo-express/discussions/1369)? Looks like you fixed a rounding bug here?: https://github.com/mongo-express/mongo-express/pull/1184

**Comment by rtritto** [comment]
> @rtritto is this fixed in the [1.0.3-rc](https://github.com/mongo-express/mongo-express/discussions/1369)? Looks like you fixed a rounding bug here?: [mongo-express/mongo-express#1184](https://github.com/mongo-express/mongo-express/pull/1184)

Yes, the fix https://github.com/mongo-express/mongo-express/pull/1184 is in version 1.0.3-rc.
